cockroachcloud/managing-cmek.md

```diff
@@ -236,11 +236,76 @@ To rotate the CMEK keys for one or more cluster regions:
       --data "@cmek_config.json"
     ```
 
+## Add a region to a multi-region CMEK-enabled cluster
+
+To add a region to a multi-region cluster, update your cluster's region definitions using the CockroachDB Cloud API [Update Cluster](https://www.cockroachlabs.com/docs/api/cloud/v1#operation/CockroachCloud_UpdateCluster) endpoint.
+
+1. Construct the data payload JSON to update you region definitions.
+
+    The payload format depends on which cloud provider you use:
+    - **AWS**:
+        {% include_cached copy-clipboard.html %}
+        ```json
+        # cmek_config.json
+        {
+            "dedicated": {
+                "region_nodes": {
+                    "us-west1": 3,
+                    "us-central1": 5
+                },
+                "cmek_region_specs": [
+                    {
+                        "region": "us-west1",
```

Comment:
nit: AWS regions have an extra dash before the number, like `us-west-1`

Reply:
thanks!